configs/predef_configs/config_AuAu_4.3_DynFlu.yaml

```
@@ -0,0 +1,38 @@
+IC:
+    Executable: /path/to/smash
+    Config_file: /path/to/handler/configs/smash_initial_conditions_DynFlu.yaml
+    Software_keys:
+        General:
+            Gaussian_Sigma: 0.7
+        Modi:
+            Collider:
+                Projectile:
+                    Particles: {2212: 79, 2112: 118} #Gold197
+                Target:
+                    Particles: {2212: 79, 2112: 118} #Gold197
+                Sqrtsnn: 4.3
+            Impact:
+                Range: [0.0,3.3] # 0-5%
+            Initial_Conditions:
+                Type: "Dynamic"
+                Energy_Density_Threshold: 0.5
+                Formation_Time_Fraction: 0.2
+                Fluidizable_Processes: ["All"]
+                Delay_Initial_Elastic: true
```

Test runs crash since indentation wrong.

```suggestion
                Impact:
                    Range: [0.0,3.3] # 0-5%
                Initial_Conditions:
                    Type: "Dynamic"
                    Energy_Density_Threshold: 0.5 
                    Formation_Time_Fraction: 0.2 
                    Fluidizable_Processes: ["All"]
                    Delay_Initial_Elastic: true
```